lib/modules/platform/bitbucket/types.ts

```diff
@@ -30,12 +30,21 @@ export interface PagedResult<T = any> {
 export interface RepoInfo {
   isFork: boolean;
   owner: string;
-  mainbranch: string;
+  mainBranch: string;
```

Comment:
please revert this unrelated change.

Reply:
sure... i'll just learn to look past the camelCase mismatch 🙈